src/renderer/components/siteinfotools/ProviderDropdown.tsx

```diff
@@ -111,8 +117,9 @@ export const ProviderDropdown = (props: Props) => {
 				{multiMachineSelect ? 'Create new site from' : 'Back up to'}
 			</span>
 			<FlyDropdown
-				className={styles.ProviderDropdown}
+				className={`${styles.ProviderDropdown} ${offline ? styles.ProviderDropdownOffline : ''}`}
```

Comment:
Can you use `classnames` here? 

Something like: 
```suggestion
				className={classnames(styles.ProviderDropdown, { [styles.ProviderDropdownOffline]: offline })}
```